public/main.js

```diff
@@ -32,5 +40,19 @@ function createWindow() {
   mainWindow.on('closed', () => {
     mainWindow = null;
   });
+
+  const poster = io.on('connection', (socket) => {
+    console.log('a user connected');
```

Comment:
Is this `console.log` just for our benefit? It’d be *really* cool if we could put it in the Electron app chrome somehow. Something like a green dot and a “Connected" label, e.g.:
<img width="819" alt="Screen Shot 2019-06-24 at 3 09 36 PM" src="https://user-images.githubusercontent.com/31315610/60045138-16989200-9692-11e9-815d-7fe2db5a1d84.png">

Reply:
I like this Idea,  We could implement that if we have time.